### ISSUE TITLE
log4brains build broken on node 14.21.3

### ISSUE THREAD

**Issue by Tenshock** [thomvaill/log4brains#106]
# Bug Report

## Description

the output generated by `log4brains build` is broken

## Steps to Reproduce

1. Install node lts/fermium v14.21.3
2. install log4brains:latest: `npm install -g log4brains`
3. initiate a log4brains project: `log4brains init`
4. build a deployable static website: `log4brains build`
5. launch the index.html at `.log4brains/out` 
6: See how the navigation, search and images are broken

## Expected Behavior

The static website should work

## Context

I can't deploy to gitlab pages or whatever using the procedure explained in the README.md

## Environment

- Log4brains version: 1.0.1
- Node.js version: 14.21.3
- OS and its version: Ubuntu 20 on WSL
- Browser information: Microsoft Edge

**Comment by theyd** [comment]
I have got the same issue with node v18.16.1
I'm running node with an environment variable: NODE_OPTION=--openssl-legacy-provider

The "Decision Log" navigation bar on the left seems empty and also the "ADR Workflow" image wouldn't be loaded.

**Comment by jasonholmberg** [comment]
I was able to get this to work by changing the last two steps in the documented GH pages workflow to:
```
      - name: Install and Build Log4brains
        run: |
          npm install -g log4brains
          log4brains build
      - name: Deploy
        uses: JamesIves/github-pages-deploy-action@v4
        with:
          branch: gh-pages
          folder: .log4brains/out
          clean: true
```
It seems like anything other that the default `buildPath` break any resource links when deployed to GH Pages.

**Comment by theyd** [comment]
At first step, i don't want to deploy the pages to Github etc.

When i run `log4brains build`, the page index.html seems not working correctly. The Logo in the Header is missing and the links on the left navigationbar are invisible. When i search for the links in the HTML file, they exist but maybe the css does hide the links. In addition i find the link to the logo in the HTML file: `/l4b-static/Log4brains-logo-dark.png`. When i edit the link and remove the first slash, the logo will be shown.

running `log4brains preview` is working fine.

Im running log4brains under Windows.

![log4brains](https://github.com/thomvaill/log4brains/assets/51789712/0f1f9f1d-4066-4a67-af8d-f6d8a514362a)

**Comment by thomvaill** [comment]
It looks like there are two different issues mentioned here:

- Issues with some NodeJS versions: normally fixed in version [1.0.2-alpha.2](https://www.npmjs.com/package/log4brains/v/1.0.2-alpha.2)
- Base path issues: don't forget to add the `--basePath /your/path` parameter to the build command when log4brains is not served at the root path
